src/libs/actions/Session/index.js

```diff
@@ -383,6 +383,43 @@ function signIn(password, validateCode, twoFactorAuthCode, preferredLocale = CON
     API.write('SigninUser', params, {optimisticData, successData, failureData});
 }
 
+function signUp(preferredLocale = CONST.LOCALES.DEFAULT) {
```

Comment:
Needs a JS Doc

Reply:
so sorry I forgot to link the related Web-E PR for this.

the notification for this weird naming is [explained on the OP of that PR](https://github.com/Expensify/Web-Expensify/pull/37507#issue-1716210807).